### ISSUE TITLE
Examples should include @language

### ISSUE THREAD

**Issue by r12a** [csarven/ldn#50]
[raised on behalf of Felix Sasaki and Addison Phillips]
The draft has a normative dependency on json-ld. One should have at least one example in the draft that shows the I18N aspects of json-ld, based e.g. on the json-ld string internationalization section https://www.w3.org/TR/json-ld/#string-internationalization

In fact we'd recommend adding language information to all examples containing natural language text, to convey the idea that it's useful information.

The example at https://www.w3.org/TR/ldn/#sending  includes an `@Type` with an apparent English language value (`"Announce"`). There is no use of the JSON-LD `@language` context value to set the document language (in this case to English. Should there be?

**Comment by rhiaro** [comment]
`Announce` is an ActivityStreams2 type, and that example uses the AS2 context, so `"@type": "Announce"` expands to `"@type": { "@id": "http://www.w3.org/ns/activitystreams#Announce" }` so it's actually a URI not an English language string.

**Comment by csarven** [comment]
There was only one (JSON-LD) example in the spec that was missing `@language` which had content; it was the SIOC example. Updated.

I've also added additional language tags to the HTML example (in discovery) - its literal content is not important or of use in the example, but added for good practice. Also added a literal with a 'hy' language tag in the JSON-LD provenance example under Payload Examples.
